ddhcpd/luasrc/lib/gluon/upgrade/500-ddhcpd

```diff
@@ -2,6 +2,7 @@
 local site = require('gluon.site')
 local uci = require('simple-uci').cursor()
 if site.ddhcpd then
+  -- TODO: this seems to be true all the time
   local enabled = site.ddhcpd.enabled() or '1'
```

Comment:
I mean this is always '1', isn't it?

Reply:
I see, I was mistaken: 

lua is special:

    > '0' or '1'
    0

genial!